pkg/cloudprovider/provider/hetzner/provider.go

```diff
@@ -223,13 +218,19 @@ func (p *provider) Validate(ctx context.Context, _ *zap.SugaredLogger, spec clus
 		}
 	}
 
-	if c.Image != "" {
-		//nolint:staticcheck // We do not have the architecture available here.
-		if _, _, err = client.Image.Get(ctx, c.Image); err != nil {
-			return fmt.Errorf("failed to get image: %w", err)
+	image := c.Image
+	if image == "" {
+		image, err = getNameForOS(pc.OperatingSystem)
```

Comment:
This `image` variable is not used later.

Reply:
You are right, I pushed a new commit.